masonry/src/passes/update.rs

```diff
@@ -77,6 +77,14 @@ pub(crate) fn run_update_pointer_pass(root: &mut RenderRoot, root_state: &mut Wi
     let pointer_pos = root.last_mouse_pos.map(|pos| (pos.x, pos.y).into());
 
     // -- UPDATE HOVERED WIDGETS --
+    // If the capture target is disabled, stashed or removed, we set
+    // the capture target id to None.
```

Comment:
```suggestion
    // Release the pointer capture if it no longer makes sense.
```
The previous comment was just exactly restating the code.